includes/classes/base.php

```diff
@@ -67,6 +67,19 @@ public function _get_val( $key ) {
 		return isset( self::$_get[ $key ] ) ? self::$_get[ $key ] : null;
 	}
 
+	/**
+	 * Returns an array of the key=>value | null in $_GET if the given keys exist.
+	 * @param $keys
+	 * @return array|null
+	 */
+	public function _get_vals($keys){
```

Comment:
```suggestion
	public function _get_vals($keys)
	{
```

The CS fixer in me was twitching 😂